atom/app/atom_content_client.cc

```diff
@@ -75,6 +77,15 @@ std::string AtomContentClient::GetProduct() const {
 void AtomContentClient::AddAdditionalSchemes(
     std::vector<std::string>* standard_schemes,
     std::vector<std::string>* savable_schemes) {
+  std::string custom_schemes;
+  scoped_ptr<base::Environment> env(base::Environment::Create());
+  env->GetVar("SCHEMES", &custom_schemes);
+  if (!custom_schemes.empty()) {
+    base::StringTokenizer schemes(custom_schemes, ",");
```

Comment:
cool, is this user-friendly enough for using environment variable for fix ? Where should i document about this btw ?